helix-term/src/args.rs

```diff
@@ -11,6 +11,8 @@ pub struct Args {
     pub load_tutor: bool,
     pub fetch_grammars: bool,
     pub build_grammars: bool,
+    pub vsplit: bool,
+    pub hsplit: bool,
```

Comment:
You could reuse the [`Layout` enum](https://github.com/helix-editor/helix/blob/a17626a822b36d4de3146c2d410f976e19dd189c/helix-view/src/tree.rs#L49-L53) here and represent it using a single `pub split: Option<Layout>`,where `None` would mean no splitting. The binary would still have both `--vsplit` and `--hsplit`, and only the parsing would change.